backend/src/main/java/org/hackbrooklyn/plaza/service/impl/AnnouncementServiceImpl.java

```diff
@@ -36,4 +40,12 @@ public AnnouncementServiceImpl(EntityManager entityManager) {
 
         return typedQuery.getResultList();
     }
+
+    @Override
+    public void createNewAnnouncements(String body, User author) {
+        Announcement announcement = new Announcement();
+        announcement.setBody(body);
+        announcement.setAuthor(author);
+        announcementRepository.save(announcement);
```

Comment:
This looks great! It would be nice if we could get the created announcement ID, construct a link to the newly created announcement, and send it back to the user as a 201 Created status.